src/Fixer/Phpdoc/PhpdocOrderFixer.php

```diff
@@ -120,21 +120,26 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
                 continue;
             }
 
+            $configurationOrder = [];
+            foreach ($this->configuration['order'] as $type) {
+                $configurationOrder[] = $type;
+                $configurationOrder[] = 'psalm-'.$type;
```

Comment:
We don't use Psalm internally, so if we want to include this subset for the upstream users, I would recommend putting it _after_ `phpstan-` subset. Not only it's a tool used by Fixer vs additional tool, but also it satisfies alphanumeric sorting too.

Reply:
> satisfies alphanumeric sorting

👍 done